src/classiceditorui.js

```diff
@@ -29,20 +29,24 @@ export default class ClassicEditorUI extends BoxedEditorUI {
 		super( editor );
 
 		/**
-		 * Toolbar controller.
+		 * Editable UI controller.
 		 *
 		 * @readonly
-		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
+		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
 		 */
-		this.toolbar = this._createToolbar();
+		this.editable = this._createEditableUI();
+	}
 
+	init() {
 		/**
-		 * Editable UI controller.
+		 * Toolbar controller.
 		 *
 		 * @readonly
-		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
+		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
 		 */
-		this.editable = this._createEditableUI();
+		this.toolbar = this._createToolbar( this.editor.ui.view.element );
```

Comment:
The element will be available immediately after toolbar is created, not initialised. So why moving it here?